### PR TITLE
2 dimensional arrays assignment - intervals problems

### PR DESCRIPTION
**Pull request by babywlfcd**

### CODE REVIEW

File: Practice/Course/Assignments/ProblemsOnSorting.cs [hunk 1]
```diff
@@ -10,6 +10,49 @@ public class ProblemsOnSorting
         // 5. Quick Sort
         // https://github.com/babywlfcd/dsa/blob/main/Practice/Sorting.cs
 
+        /// <summary>
+        /// Question - 1 - merge all overlapping intervals
+        /// Solution: in place implementation
+        ///     1. keep tracking 2 pointers for traverse and remove in place the overlap interval
```
Comment (christianlup) [inline Practice/Course/Assignments/ProblemsOnSorting.cs:16]: keep 2 tracking pointers

File: Practice/Course/Assignments/ProblemsOnSorting.cs [hunk 2]
```diff
@@ -37,6 +80,33 @@ public bool CanAttendToMeetings(List<int[]> input)
...
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public int CalculateConferenceRooms(List<int[]> input)
+        {
+
+            if (input.Count == 0)
+                return 0;
+
+            var countOverlap = 1;
```
Comment (christianlup) [inline Practice/Course/Assignments/ProblemsOnSorting.cs:98]: Why we start at 1? should be zero

Comment (christianlup) [inline Practice/Course/Assignments/ProblemsOnSorting.cs:98]: Why we start at 1? should be zero

**Review by christianlup — CHANGES_REQUESTED** [review]

File: Practice/Course/Assignments/ProblemsOnSorting.cs (hunk 2 above)
Comment (babywlfcd) [reply Practice/Course/Assignments/ProblemsOnSorting.cs:98]: accommodate a meeting room for first interval

**Review by CarmenCD — APPROVED** [review]

**Review by CarmenCD — APPROVED** [review]

**Pull request closed without merging**